### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes are to be made

Files to Check -

- [ ] src/App.js
- [ ] src/backend/db/categories.js
- [ ] src/components/HomePage/HomeBottomContainer/HomePageContainer.jsx
- [ ] src/components/HomePage/HomePageCard/HomePageCard.jsx
- [ ] src/components/HomePage/HomePageHero/HomePageHero.jsx
- [ ] src/components/Nav/Nav.jsx
- [ ] src/pages/home/HomePage.jsx
- [ ] src/pages/landing/LandingPage.jsx

home_page

### CODE REVIEW

File: src/backend/db/categories.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import { v4 as uuid } from "uuid";
 
+import grouptherapy from "../../assets/home_page_images/image_gt.jpg";
+import clothing from "../../assets/home_page_images/image_lane8_model.jpg";
+import vinyl from "../../assets/home_page_images/image_hp_vinyl.png";
+import poster from "../../assets/home_page_images/poster.png";
+import cds from "../../assets/home_page_images/image_cd.png";
+import newarrivals from "../../assets/home_page_images/bottom_1.jpg";
+import trending from "../../assets/home_page_images/bottom_2.jpg";
```
Comment (viru2001) [inline src/backend/db/categories.js:9]: you can make file index.js in the assets folder ... from there you can export all your images ... so while importing them in categories.js you can import all images from one file

File: src/components/HomePage/HomeBottomContainer/HomePageContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                <p className="horizontal-card-title">Trending</p>
+                <small>
+                  Viral, trending & taking off! A combination of top chart &
+                  viral trends, all your favorites in one place.
+                </small>
+              </div>
+            </div>
+          </div>
+        </div>
+      </Link> */}
```
Comment (viru2001) [inline src/components/HomePage/HomeBottomContainer/HomePageContainer.jsx:64]: remove unused code

File: src/components/index.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import {Nav} from "./Nav/Nav.jsx"
+import {HomePageCard} from "./HomePage/HomePageCard/HomePageCard"
+import {HomePageHero} from "./HomePage/HomePageHero/HomePageHero"
+import {HomePageContainer} from "./HomePage/HomeBottomContainer/HomePageContainer"
```
Comment (viru2001) [inline src/components/index.js:4]: here also you can export all files from HomePage in one file

File: src/pages/home/homePage.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
+@import url("https://fonts.googleapis.com/css2?family=Akaya+Telivigala&family=Indie+Flower&family=Montserrat:wght@300&family=Nunito+Sans:wght@700&family=Open+Sans+Condensed:wght@700&family=Shadows+Into+Light&display=swap");
+
```
Comment (viru2001) [inline src/pages/home/homePage.css:2]: you can import this in your index.html

**Review by viru2001 — APPROVED** [review]
Code and folder structure is good.  I liked you have made a separate file for defining Routes. I gave few suggestions.

**Review by Rishi05051997 — COMMENTED** [review]
Please do changes as per suggested by Viresh & then good to go

**Pull request merged**